networking/network_observability/installing-operators.adoc

```diff
@@ -5,26 +5,35 @@ include::_attributes/common-attributes.adoc[]
 :context: network_observability
 
 toc::[]
-Installing Loki is a prerequisite for using the Network Observability Operator. It is recommended to install Loki using the Loki Operator; therefore, these steps are documented below prior to the Network Observability Operator installation.
+Installing Loki is a recommended prerequisite for using the Network Observability Operator. You can choose to use xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki], but there are some considerations for doing this, described in the previous linked section.
 
-The Loki Operator integrates a gateway that implements multi-tenancy & authentication with Loki for data flow storage. The *LokiStack* resource manages *Loki*, which is a scalable, highly-available, multi-tenant log aggregation system, and a web proxy with {product-title} authentication. The *LokiStack* proxy uses {product-title} authentication to enforce multi-tenancy and facilitate the saving and indexing of data in *Loki* log stores. 
+The Loki Operator integrates a gateway that implements multi-tenancy & authentication with Loki for data flow storage. The `LokiStack` resource manages Loki, which is a scalable, highly-available, multi-tenant log aggregation system, and a web proxy with {product-title} authentication. The `LokiStack` proxy uses {product-title} authentication to enforce multi-tenancy and facilitate the saving and indexing of data in Loki log stores. 
 
 [NOTE]
 ====
 The Loki Operator can also be used for xref:../../logging/cluster-logging-loki.adoc#cluster-logging-loki[Logging with the LokiStack]. The Network Observability Operator requires a dedicated LokiStack separate from Logging.
 ====
 
+include::modules/network-observability-without-loki.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* For more information about exporting flow data to Kafka for third party processing consumption, see xref:../../networking/network_observability/configuring-operator.adoc#network-observability-enriched-flows-kafka_network_observability[Export enriched network flow data].
```

Comment:
I believe latest [repo guidelines on additional resources lists](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) wants us to avoid sentences in the bullets, and just have the link / name of section.